core-feature-pack/src/main/resources/content/bin/domain.conf

```diff
@@ -48,13 +48,14 @@ fi
 if [ "x$JAVA_OPTS" = "x" ]; then
    JAVA_OPTS="-Xms64m -Xmx512m -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true"
    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS -Djava.awt.headless=true"
+
+   # Use JBoss Modules lockless mode
```

Comment:
Moving of these commented JAVA_OPTS settings is not described in [jira](https://issues.jboss.org/browse/WFCORE-3825) or in description of this PR. These changes could cause the regression of behaviour. I recommend to remove all changes of domain.conf file.